packages/scandipwa/src/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -176,7 +176,12 @@ export class MyAccountDispatcher {
             /** @namespace Store/MyAccount/Dispatcher/confirmAccountFetchMutationThen */
             () => dispatch(showNotification('success', __('Your account is confirmed!'))),
             /** @namespace Store/MyAccount/Dispatcher/confirmAccountFetchMutationError */
-            () => dispatch(showNotification('error', __('Something went wrong! Please, try again!')))
+            (err) => {
+                const { message } = err[0] || err;
+                return dispatch(
+                    showNotification('error', message || __('Something went wrong! Please, try again!'))
```

Comment:
Prefer default values for destructured variables, where it is possible (`const {a=1} = options`)